google/cloud/resourcemanager/v2/folders.proto

```diff
@@ -201,11 +207,11 @@ message Folder {
 
   // Output only. The resource name of the Folder.
   // Its format is `folders/{folder_id}`, for example: "folders/1234".
-  string name = 1;
+  string name = 1 [(google.api.resource_definition).path = "folders/{folder}"];
```

Comment:
So this assumes that each message type can have at most one `resource_definition`?